src/readabilityScore/TextFileReader.java

```diff
@@ -0,0 +1,26 @@
+package readabilityScore;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class TextFileReader {
+
+    private String fileContent;
```

Comment:
Usually it's better to avoid storing a state in an instance fields if it's possible.

The reason is that if we have a mutable state (instance fields) then it's easier to make some bugs...

It looks like we could have a public method that accepts a file name and returns the file content as String.
This way we don't need to store any state and create a separate instance of `TextFileReader` each time we need to read a text from file